index.html

```diff
@@ -10,6 +10,21 @@
 <body>
   <header>
     <a href="#" id="my-logo">Farida's Logo</a>
+    <img src="images/home.png" alt="description" id="home"/>
   </header>
-      </body>
+
+  <section id="intro">
+    <section id="text">
+      <h1>Farida Hamid</h1>
+      <p>Software Engineer and consultant to educational platform since March 2020.</p>
+      <h3>let's connect</h3>
+      <ul id="contacts">
+        <li><a href="#"><img src="images/vec.png" alt="twitter"></a></li>
+        <li><a href="#"><img src="images/Linkedin-icon.png" alt="Linkedin-icon"></a></li>
+        <li><a href="#"><img src="images/vec-3.png" alt="email"></a></li>
+        <li><a href="#"><img src="images/vec-2.png" alt="github"></a></li>
+      </ul>
+    </section>
+  </section>
+</body>
```

Comment:
- [Optional  ] @Farida-Hamid Nice work by working with the project but please make your website responsive for screens with 320px width
- 338px view
![image](https://user-images.githubusercontent.com/74744787/159122895-829e5d45-0282-491f-acb5-135ebbdfcb30.png)

- 320 px view
- 
![image](https://user-images.githubusercontent.com/74744787/159122947-b4e293c8-f91d-441b-b406-3164780f2b37.png)